src/create-dockerfile

```diff
@@ -234,5 +235,8 @@ COPY opam/install-opam-packages .ocaml-setup/opam/install-opam-packages
 RUN $(multiline ./.ocaml-setup/opam/install-opam-packages $opam_packages)
 $([[ "$no_cmd" = true ]] || (echo; echo "CMD $cmd") )
 
+# Some final steps to run before we finish.
+RUN $(multiline ./.ocaml-setup/finalize)
```

Comment:
I would place the COPY instruction just above this one because they go together.